src/main/java/edu/ucsd/sbrg/bigg/ModelPolisher.java

```diff
@@ -73,6 +74,10 @@
      * @see ModelPolisherOptions#ANNOTATE_WITH_BIGG
      */
     Boolean annotateWithBiGG = null;
+    /**
+     * @see ModelPolisherOptions#OUTPUT_COMBINE
+     */
+    Boolean outputCOMBINE = null;
```

Comment:
Not sure if it is a good idea to set this to null; This should either be `true` or `false` as default option (same for the annotateWithBigg flag above).

Reply:
Especially because the argument parser sets these be default to False, so here the same defaults should be used.

Reply:
I opened a separate issue for this (see #66), this is probably not part of the COMBINE archive feature, but should be done.

Reply:
All the flags / command line options are set to null initially. We can set all of them to false initially, but that's not required as we already set them to true/false while getting arguments from the command line (default being false as you said). 
Also, I think if one misses getting an argument from cmd-line then NullPointer will be hellpful in debugging.

Reply:
Around line 80, these are just declarations of various parameters, they are instantiated in method `initParameters`. So, it is fine to keep parameters = null in the declaration.

Reply:
I am fine with `null`. In my opinion makes things more readable to see the defaults in the initialization. Also it decouples things from the argument parser, i.e, write now I need the argument parser so that the variables get usable values. But I can just look things up in the argument parser. Like stated above I am fine with that (you don't have to write `= null`, just leave the variables uninitialized)